src/lib.rs

```diff
@@ -1738,6 +1741,13 @@ ffi!(fn pgp_export_keydata(session: *mut Session,
 
     let mut keydata = Vec::new();
     if secret {
+        // No-op if there is no passphrase (NULL).
+        // If there is a passphrase, only encrypted parts are decrypted.
```

Comment:
```suggestion
```

What else, if not encrypted packets, would you decrypt? :) -> useless comment, remove

Reply:
IMO this is only clear if you are an insider re certificates. Hence the comment. I want to stress that decrypting something that has not been encrypted in the first place is not an error.

Reply:
Hm okay. Maybe this comment I [proposed above](https://github.com/plancksecurity/foundation-planckCoreSequoiaBackend/pull/15#discussion_r1716875579) is clearer and can replace this comment?

Alternatively, a clearer version would be:

> If there is a passphrase but the cert is unencrypted, this is also a no-op.

Reply:
Will try to clarify this. I don't consider it obvious at all that you can decrypt something that was not encrypted in the first place. E.g. if you decrypt an unencrypted text file, you'll get garbage.

Reply:
> E.g. if you decrypt an unencrypted text file, you'll get garbage.

True, and I agree it's worth to document it. Just trying to find a better phrasing, since I needed your explanation here to understand the comment.